### DISCUSSION TITLE
Добавление выбора маркетов: spot, futures и др.

### DISCUSSION THREAD

**Discussion by khanbekov** [RoboTradeCode/configurator#36] [Ideas]
Создано на базе #34 
> В binance есть разные секции, spot, coin-m и usm-s, и возможно на всех этих секциях разные названия валютных пар, и я думаю, что для okx есть тоже самое.
> т.е. в идеале сделать так, что бы конфигуратор работал так:
> `/okx/1` - спот секция
> `/okx-coinm/1` - coinm секция и т.д.

>Такое можно сделать, но я думаю, лучше сделать это отдельным параметром запроса: `/binance/1/?market_type=futures`

**Comment by 81xx** [comment]
Да, такой вариант наверное более удобен, но тогда все варианты секций для каждой биржы должны быть уникальными, т.е. надо сделать так, что бы эти типы были одинаковы и для oxk и binance.
Давай поступим так, пока что такой функционал не нужен. Сделай, что бы по умолчанию была секция `spot`. Когда нужны будут другие секции вернемся к такому функционалу.

**Reply by khanbekov** [reply]
Да, про варианты секций для каждой биржи верное замечание. С другой стороны, можно сделать разные инстансы под разные типы маркетов?

Сейчас так и есть, конфигуратор отдает только `spot`.

**Reply by 81xx** [reply]
принял, спасибо.
